src/ServiceControl.Monitoring/Http/QueueLengthApiModule.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace ServiceControl.Monitoring.Http
+{
+    using System.Linq;
+    using Nancy;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
+    using QueueLength;
+
+    public class QueueLengthApiModule : NancyModule
+    {
+        /// <summary>
+        /// Initializes the metric API module.
+        /// </summary>
+        public QueueLengthApiModule(QueueLengthDataStore store) : base("/metrics")
```

Comment:
This is a result of refactoring the bits that were already there. I propose to leave this as is and make proper changes when we start delivering features around `QueueLength`.